setuptools/tests/environment.py

```diff
@@ -18,6 +18,13 @@ class VirtualEnv(jaraco.envs.VirtualEnv):
     def run(self, cmd, *args, **kwargs):
         cmd = [self.exe(cmd[0])] + cmd[1:]
         kwargs = {"cwd": self.root, **kwargs}  # Allow overriding
+        # If no environment was passed as an argument
+        # create a copy of the current env to use when spawning a subprocess
+        # Unset PYTHONPATH - tests in venv need only that venv's paths in sys.path
```

Comment:
I think it would be nice to summarise the PR discussion in this comment and mention for which reason PYTHONPATH would be set in the first place, and how it affects the tests...